components/raftstore/src/store/fsm/peer.rs

```diff
@@ -3978,8 +3910,11 @@ where
             size += sst.total_bytes;
             keys += sst.total_kvs;
         }
-        self.fsm.peer.approximate_size = Some(self.fsm.peer.approximate_size.unwrap_or(0) + size);
-        self.fsm.peer.approximate_keys = Some(self.fsm.peer.approximate_keys.unwrap_or(0) + keys);
+        self.fsm.peer.approximate_size = self.fsm.peer.approximate_size + size;
+        self.fsm.peer.approximate_keys = self.fsm.peer.approximate_keys + keys;
```

Comment:
If you want to check the size again, you don't need to see the approximate size here

Reply:
And if that `has_calculated_region_size` is unnecessary, we can just keep the approximate be `Option`

Reply:
Emmmm, but we will report to PD at once and TiKV will not schedule split-check during importing.